src/common/constants.ts

```diff
@@ -1,2 +1,7 @@
 export const constructorName = 'constructor';
 export const yamlHeader = '### YamlMime:UniversalReference';
+export const enum topLevelItems {
+    CLASS = 'class',
```

Comment:
Why all upper cases? This is not common in JavaScript.

Reply:
yes, It should be upper the first character.